docs/quickstart.rst

```diff
@@ -226,7 +226,7 @@ Retrieve a single application
     response = client.get_application(uuid)
 
 Docs:
-`https://developer.nexmo.com/api/application#retrieve-an-application <https://developer.nexmo.com/api/application#retrieve-an-application>`__
+`https://developer.vonage.com/api/application#retrieve-an-application <https://developer.vonage.com/api/application#retrieve-an-application>`__
```

Comment:
This needs correct to point to the developer docs, this URL isn't valid. `docs.nexmo.com` is also just a redirect, so this should point to the current URL.